reporter/src/funTest/kotlin/reporters/SpdxDocumentReporterTest.kt

```diff
@@ -89,3 +113,139 @@ private fun patchExpectedResult(expectedResultFile: String, actualSpdxDocument:
             "<REPLACE_DOCUMENT_NAMESPACE>" to actualSpdxDocument.documentNamespace
         )
     )
+
+private fun createOrtResult(): OrtResult {
+    val analyzedVcs = VcsInfo(
+        type = VcsType.GIT,
+        revision = "master",
+        resolvedRevision = "10203040",
+        url = "https://github.com/path/first-project.git",
+        path = ""
+    )
+
+    return OrtResult.EMPTY.copy(
+        repository = Repository(
+            vcs = analyzedVcs,
+            vcsProcessed = analyzedVcs
+        ),
+        analyzer = AnalyzerRun(
+            environment = Environment(),
+            config = AnalyzerConfiguration(ignoreToolVersions = true, allowDynamicVersions = true),
+            result = AnalyzerResult(
+                projects = sortedSetOf(
+                    Project(
+                        id = Identifier("Maven:first-project-group:first-project-name:0.0.1"),
+                        declaredLicenses = sortedSetOf("MIT"),
+                        definitionFilePath = "",
+                        homepageUrl = "first project's homepage",
+                        scopes = sortedSetOf(),
+                        vcs = analyzedVcs
+                    )
+                ),
+                packages = sortedSetOf(
+                    CuratedPackage(
+                        pkg = Package(
+                            id = Identifier("Maven:first-package-group:first-package:0.0.1"),
+                            binaryArtifact = RemoteArtifact("https://some-host/first-package.jar", Hash.NONE),
+                            declaredLicenses = sortedSetOf("BSD-3-Clause", "MIT OR GPL-2.0-only"),
+                            description = "A package with all supported attributes set, with a VCS URL containing a" +
+                                    "user name, and with a scan result containing two copyright finding matched to a " +
+                                    "license finding.",
+                            homepageUrl = "first package's hompage URL",
```

Comment:
Spelling mistake in "hompage". Prefer to have this as a real URL e.g. https://example.com

Reply:
Fixed the spelling.